mage/releases/build.go

```diff
@@ -73,10 +74,17 @@ func XBuild(pkg string, name string, binDir string, goos string, goarch string)
 func XBuildAll(pkg string, name string, binDir string) {
 	var g errgroup.Group
 	for _, goos := range supportedClientGOOS {
-		goos := goos
-		g.Go(func() error {
-			return XBuild(pkg, name, binDir, goos, "amd64")
-		})
+		for _, arch := range supportedClientArch {
+			goos := goos
+			arch := arch
+			// skip windows arm64
+			if arch == "arm64" && goos == "windows" {
```

Comment:
In Go 1.17 we should be able to remove this skip here. Can you add a comment about that and link to the issue for Go windows/arm64 support? That way we are more likely to remember. 😀 

https://github.com/golang/go/issues/36439